sdk/apimanagement/Azure.ResourceManager.ApiManagement/api/Azure.ResourceManager.ApiManagement.netstandard2.0.cs

```diff
@@ -883,7 +883,7 @@ public partial class ApiManagementNamedValueData : Azure.ResourceManager.Models.
         public string DisplayName { get { throw null; } set { } }
         public bool? IsSecret { get { throw null; } set { } }
         public Azure.ResourceManager.ApiManagement.Models.KeyVaultContractProperties KeyVaultDetails { get { throw null; } set { } }
-        public System.Collections.Generic.IList<string> Tags { get { throw null; } }
+        public System.Collections.Generic.IList<string> Tags { get { throw null; } set { } }
```

Comment:
Why we added setters for the collection types? This breaks our design guidance I believe.